### PR TITLE
📝 Overhauling The Documentation

### PR DESCRIPTION
**Pull request by RPSMain**
### Changes

-   `improvements`: Docs

 ### Check off the following

-   [x] I have tested my changes with the current requirements
-   [x] My Code follows the pep8 code style.

### CODE REVIEW

File: docs/api/pincer.core.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@
 .. currentmodule:: pincer.core
 
 Pincer Core Module
-==================
+-------------------
```
Comment (ooliver1) [inline docs/api/pincer.core.rst:5]: MMM HH why, `=` is a top level title

File: docs/api/pincer.middleware.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .. currentmodule:: pincer.middleware
 
 Pincer Middleware Module
-========================
+----------------------------
```
Comment (ooliver1) [inline docs/api/pincer.middleware.rst:4]: same again, `=` is a top level title

File: docs/api/pincer.objects.rst [hunk 3]
```diff
@@ -0,0 +1,507 @@
+.. currentmodule:: pincer
+
+Pincer Objects Module
```
Comment (ooliver1) [inline docs/api/pincer.objects.rst:3]: mmmh these would all be the same size titles, `=` `-` and `~` are there to separate it

File: docs/api/pincer.objects.rst [hunk 4]
```diff
@@ -0,0 +1,507 @@
+.. currentmodule:: pincer
```
Comment (ooliver1) [inline docs/api/pincer.objects.rst:1]: why current module then you still do ``pincer.objects.app.Application``

**Comment by ooliver1** [comment]
i know its a draft but so far they dont look like useful changes

**Comment by Lunarmagpie** [comment]
I agree with Ooliver on this

**Pull request closed without merging**